ssl.go

```diff
@@ -64,13 +68,13 @@ func ssl(o values) func(net.Conn) net.Conn {
 // in the user's home directory. The configured files must exist and have
 // the correct permissions.
 func sslClientCertificates(tlsConf *tls.Config, o values) {
-	sslkey := o.Get("sslkey")
-	sslcert := o.Get("sslcert")
+	sslkey, sslkeyOK := o["sslkey"]
+	sslcert, sslcertOK := o["sslcert"]
 
 	var cinfo, kinfo os.FileInfo
 	var err error
 
-	if sslcert != "" && sslkey != "" {
+	if sslkeyOK && sslcertOK {
```

Comment:
As I understand it, this change breaks compatibility with `libpq` connection strings.

If I'm reading `libpq` source correctly, these settings fallback to the user/home paths when the supplied value is blank:

- [`if (conn->sslcert && strlen(conn->sslcert) > 0) ... else ... "%s/%s", homedir, USER_CERT_FILE`](https://github.com/postgres/postgres/blob/REL9_6_2/src/interfaces/libpq/fe-secure-openssl.c#L1036-L1039)
- [`if (conn->sslkey && strlen(conn->sslkey) > 0) ... else ... "%s/%s", homedir, USER_KEY_FILE`](https://github.com/postgres/postgres/blob/REL9_6_2/src/interfaces/libpq/fe-secure-openssl.c#L1219-L1222)